src/assets/scripts/uio.js

```diff
@@ -23,3 +26,5 @@ var uio = uio || {};
         });
     };
 })(jQuery, fluid);
+
+uio.setupUIO('/');
```

Comment:
Seems a bit overkill to have the `uio.setupUIO` function and always call it with the same path. It would be easier to just include `fluid.uiOptions.prefsEditor` with the proper config directly.